GraphWebApi/Controllers/TourStepsController.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Linq;
+using GraphWebApi.Common;
+using Microsoft.ApplicationInsights.DataContracts;
+using UtilityService;
+using TourStepsService.Interfaces;
+using TourStepsService.Models;
+using Microsoft.ApplicationInsights;
+
+namespace GraphWebApi.Controllers
+{
+    [ApiController]
+    public class TourStepsController : ControllerBase
+    {
+        private readonly ITourStepsStore _tourStepsStore;
+        private readonly Dictionary<string, string> _tourStepsTraceProperties =
+            new()
+            {
+                {
+                    UtilityConstants.TelemetryPropertyKey_TourSteps,
+                    nameof(TourStepsController)
+                }
+            };
+        private readonly TelemetryClient _telemetryClient;
+        public TourStepsController(ITourStepsStore tourStepsStore, TelemetryClient telemetryClient)
+        {
+            UtilityFunctions.CheckArgumentNull(tourStepsStore, nameof(tourStepsStore));
+            UtilityFunctions.CheckArgumentNull(telemetryClient, nameof(telemetryClient));
+            _tourStepsStore = tourStepsStore;
+            _telemetryClient = telemetryClient;
+        }
+
+        //Gets list of all steps
+        [Route("api/graphexplorersteps")]
+        [Route("toursteps")]
+        [Produces("application/json")]
+        [HttpGet]
+        public async Task<IActionResult> GetTourStepsListAsync(string org, string branchName)
```

Comment:
```suggestion
        public async Task<IActionResult> GetTourStepsListAsync(string org = default, string branchName = default)
```

so people can call the endpoint without providing values for those two parameters